src/components/LibraryDetailPage.tsx

```diff
@@ -99,9 +99,10 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
       "Contact & URLs": "urls_and_contact",
       "Areas": "areas"
     };
-    Object.entries(categories).filter(([k, v]) => library[v]).forEach(([k, v]) => {
+    Object.entries(categories).forEach(([k, v]) => {
+    // Object.entries(categories).filter(([k, v]) => library[v]).forEach(([k, v]) => {
       let category = library[v];
-      let categoryItems: any[] = Object.values(category);
+      let categoryItems: (string | string[])[] = Object.values(category);
```

Comment:
In what case will this return an array of array of strings?

Reply:
The geographical data.  (Because you can have multiple locations per value.)